src/widgets-service.js

```diff
@@ -36,7 +36,8 @@ WidgetsService.prototype.newCategoryModel = function (attrs, layer) {
   var dataviewModel = this._dataviews.createCategoryModel(layer, {
     type: 'category',
     column: attrs.column,
```

Comment:
I think `column` is the name of the column that is used to get the names of the categories, while `aggregationColumn` is the column that is used to aggregate results by.